test/packetimpact/tests/BUILD

```diff
@@ -6,11 +6,12 @@ package(
 )
 
 packetimpact_go_test(
-    name = "fin_wait2_timeout",
-    srcs = ["fin_wait2_timeout_test.go"],
+    name = "syn_with_same_ip",
+    srcs = ["syn_with_same_ip_test.go"],
```

Comment:
Don't remove the old test, just add a new one.

Reply:
[GES] Ok taken care.